Tests/Marketplace/marketplace_services.py

```diff
@@ -1099,15 +1100,21 @@ def upload_integration_images(self, storage_bucket):
                 image_name = os.path.basename(image_path)
                 image_storage_path = os.path.join(pack_storage_root_path, image_name)
                 pack_image_blob = storage_bucket.blob(image_storage_path)
-                print(f"Uploading {self._pack_name} pack integration image: {image_name}")
 
+                print(f"Uploading {self._pack_name} pack integration image: {image_name}")
                 with open(image_path, "rb") as image_file:
                     pack_image_blob.upload_from_file(image_file)
-                    uploaded_integration_images.append({
-                        'name': image_data.get('display_name', ''),
-                        'imagePath': urllib.parse.quote(pack_image_blob.name) if GCPConfig.USE_GCS_RELATIVE_PATH
-                        else pack_image_blob.public_url
-                    })
+
+                if GCPConfig.USE_GCS_RELATIVE_PATH:
+                    image_gcs_path = urllib.parse.quote(
+                        os.path.join(GCPConfig.IMAGES_BASE_PATH, self._pack_name, image_name))
+                else:
+                    image_gcs_path = pack_image_blob.public_url
+
+                uploaded_integration_images.append({
+                    'name': image_data.get('display_name', ''),
+                    'imagePath': image_gcs_path
+                })
```

Comment:
when do you need this part?

Reply:
In pack metadata

Reply:
@yuvalbenshalom you mean ..?
```
image_gcs_path = pack_image_blob.public_url
```

Reply:
Yes, when will you not use the relative path? and why?

Reply:
Currently we use only relative path. At the past we used full public url. It is configurable in case we want for some reason to switch back. Currently not in use...

Reply:
Only when we switch `USE_GCS_RELATIVE_PATH` to False